### PR TITLE
Add gke-alpha-features-release-1-7 test job.

### PR DESCRIPTION
**Pull request by dchen1107**

### CODE REVIEW

File: jobs/config.json [hunk 1]
```diff
@@ -4212,6 +4212,20 @@
...
+      "--env-file=jobs/platform/gke.env",
+      "--env-file=jobs/ci-kubernetes-e2e-gke-alpha-features-release-1-7.env",
+      "--timeout=180m",
+      "--extract=ci/latest-1.7",
+      "--check-leaked-resources",
+      "--test_args=--ginkgo.focus=\\[Feature:(ExternalTrafficLocalOnly|DynamicKubeletConfig)\\]|Initializers"
+    ],
+    "scenario": "kubernetes_e2e",
+    "sigOwners": [
+      "UNKNOWN"
```
Comment (krzyzacy) [inline jobs/config.json]: will some sig own those alpha-features job?

Comment (dchen1107) [inline jobs/config.json]: I leave it unknown for now because it is gke specific test. It should eventually owned by gke release manager.

**Comment by krzyzacy** [comment]
run `bazel run //jobs:config_sort` and we can merge this.

File: prow/config.yaml [hunk 2]
```diff
@@ -8853,6 +8853,39 @@ periodics:
         defaultMode: 256
         secretName: ssh-key-secret
 
+- interval: 1h
+  name: ci-kubernetes-e2e-gke-alpha-features-release-1-7
+  spec:
+    containers:
+    - args:
+      - --timeout=170
```
Comment (krzyzacy) [inline prow/config.yaml]: --timeout=200

**Pull request merged**